miden-lib/asm/miden/note.masm

```diff
@@ -109,3 +109,21 @@ export.get_sender
     syscall.get_note_sender
     # => [sender]
 end
+
+#! Returns the note args of the note currently being processed. Panics if a note is not being
+#! processed or if no note args for the note id were provided via the advice map.
+#! 
+#! NOTE_ARGS are being stored on the advice map with { NOTE_ID | NOTE_ARG} per note 
+#!
+#! Inputs: []
+#! Outputs: [NOTE_ARGS]
+export.get_note_args
```

Comment:
I'm not sure we need this procedure.

Reply:
If we put note args directly on top of the stack at execution we don't need it indeed.